pingpong/msg_pingpong.c

```diff
@@ -122,31 +136,37 @@ static int sync_test(void)
 	}
 	credits = max_credits;
 
-	ret = opts.dst_addr ? send_xfer(16) : recv_xfer(16);
+	/* In order to not be included in the data integrity check, pass iter=-1 */
+	ret = opts.dst_addr ? send_xfer(16, -1) : recv_xfer(16, -1);
```

Comment:
The use of iter for this is odd. I don't think we care if the sync messages are validated as well.  Maybe avoid adding any additional arguments into send/recv_xfer by just using a static variable inside the format buffer function. This will make it easier to move the send_xfer/recv_xfer calls into common code.